src/image.c

```diff
@@ -9667,14 +9667,6 @@ x_kill_gs_process (Pixmap pixmap, struct frame *f)
 
 #ifdef GLYPH_DEBUG
 
```

Comment:
Yeah, I missed that. How do cfg options work with `lisp_fn`? Are there any examples?

Reply:
This is new.

Reply:
I do not believe you can directly use a `cfg` with a lisp_fn macro. We will likely need to have a blah_debug.rs file that is conditionally loaded in lib.rs like how the dired support is handled.

Reply:
https://github.com/remacs/remacs/issues/1379

Reply:
Could we potentially add a cfg argument to `lisp_fn` that, if supplied, adds a cfg clause to the function?

Reply:
Good question @agraven. I am not certain. Thus far we have managed with conditionally importing files. dired is a good example of this.

Reply:
@agraven I think this would be a good solution and we should open an issue.

For now, I think it's ok to leave this PR as it is. There are only two lisp functions in this file, that depend on `GLYPH_DEBUG`.